backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1102 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Accessibility {
+  id           String        @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt    DateTime      @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt    DateTime      @updatedAt @map("updated_at") @db.Timestamp(6)
+  mobility     Boolean?
+  vision       Boolean?
+  hearing      Boolean?
+  applications Applications?
+
+  @@map("accessibility")
+}
+
+model ActivityLog {
+  id           String        @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt    DateTime      @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt    DateTime      @updatedAt @map("updated_at") @db.Timestamp(6)
+  module       String        @db.VarChar
+  recordId     String        @map("record_id") @db.Uuid
+  action       String        @db.VarChar
+  metadata     Json?
+  userId       String?       @map("user_id") @db.Uuid
+  userAccounts UserAccounts? @relation(fields: [userId], references: [id], onUpdate: NoAction)
+
+  @@map("activity_log")
+}
+
+// Note: [place_name, city, county, state, street, street2, zip_code] formerly had max length of 64 characters
+model Address {
+  id                           String            @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt                    DateTime          @default(now()) @map("created_at") @db.Timestamp(6)
```

Comment:
so those types are generated by prisma itself to map our database structure, I don't think it would cause any differences since thats the prisma way of getting us the exact same db column